canon_host/src/disk_store.rs

```diff
@@ -4,43 +4,27 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use std::fmt;
 use std::sync::Arc;
 
 use parking_lot::RwLock;
 
 use appendix::Index;
 
-use canonical::{
-    ByteSink, Canon, DrySink, Id32, InvalidEncoding, Sink, Source, Store,
-};
-use canonical_derive::Canon;
+use canonical::{Canon, CanonError, Id, IdBuilder, Sink};
 
 use std::fs::{create_dir, File, OpenOptions};
 use std::io::{self, Read, Seek, SeekFrom, Write};
 use std::path::PathBuf;
-use tempfile::tempdir;
 
 pub struct DiskStoreInner {
-    index: Index<Id32, u64>,
+    index: Index<Id, u64>,
```

Comment:
I thought the `DiskStore` is not implemented anymore due the nature of the "singleton" Store; shouldn't be removed?